src/components/Admin/AdminTabs.tsx

```diff
@@ -0,0 +1,93 @@
+        },
+        tabitem: {
+            base: 'flex items-center justify-center p-4 rounded-t-lg text-sm font-medium first:ml-0 disabled:cursor-not-allowed disabled:text-gray-400 disabled:dark:text-gray-500',
+            styles: {
+                default: {
+                    base: 'rounded-t-lg',
+                    active: {
+                        on: 'text-orange-600 rounded-t-lg border-b-2 border-orange-600 active dark:text-orange-500 dark:border-orange-500',
+                        off: 'border-b-2 border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-600 dark:text-gray-400 dark:hover:text-gray-300',
+                    },
+                },
+            },
+            icon: 'mr-2 h-5 w-5',
+        },
+    },
+    tabpanel: 'py-3',
+}
+export default function AdminTabs() {
+    const { isLoading, isError, data } = useQuery(
+        'resources',
+        getResourcesWithTags,
+        {
+            staleTime: defaultStaleTime,
+        }
+    )
+
+    if (isLoading) {
+        return <LoadingPage />
+    }
+    if (isError) {
+        return <ErrorPage />
+    }
+    return (
+        <div className="min-h-screen bg-orange-50 px-12">
+            {data && data.data ? (
+                <Tabs aria-label="Admin tabs" theme={tabsTheme}>
+                    <Tabs.Item
+                        active
+                        title="Awaiting Approval"
+                        icon={FaExclamationTriangle}
+                    >
+                        {data.data
+                            // @ts-expect-error: https://github.com/supabase/postgrest-js/pull/499
+                            .filter((r) => r.in_review)
+                            // @ts-expect-error: https://github.com/supabase/postgrest-js/pull/499
+                            .map((resource: IResource) => {
+                                return (
+                                    <UnaprovedCard
+                                        resource={resource}
+                                        key={resource.id}
+                                    />
+                                )
+                            })}
+                    </Tabs.Item>
+                    <Tabs.Item title="Approved" icon={FaCheckCircle}>
+                        {data.data
+                            // @ts-expect-error: https://github.com/supabase/postgrest-js/pull/499
+                            .filter((r) => !r.in_review)
+                            // @ts-expect-error: https://github.com/supabase/postgrest-js/pull/499
+                            .map((resource: IResource) => {
+                                return (
+                                    <ApprovedCard
+                                        resource={resource}
+                                        key={resource.id}
+                                    />
+                                )
+                            })}
+                    </Tabs.Item>
```

Comment:
We could reduce some code doing something like `data.data.reduce()` and add each status to a new object(s) we can iterate over here

Reply:
I think maybe even just one loop? And skip the filter?

```js
Data.data.map((resource) => {
  Return resource.is_approved ? <Approved/> : <Unapproved/>
}) 
```

Reply:
I can do that with a use effect and some state variables. The difficulty with the single loop in the JSX is that the data needs to go in two different tabs.